src/SyntheticMixedBenchmark/README.md

```diff
@@ -0,0 +1,79 @@
+
+With the attached scripts, DASHBOARD queries are submitted as 16 concurrent users with 2 secs wait, REPORT queries in 6 parallel streams with 15 mins wait, 4 parallel streams of DATASCIENCE queries with 30 mins wait. We have the mix dataset of both tpc-h 100g and tpc-h 3t. The smaller tpc-h 100g represents a datamart and the larger tpc-h 3t is the main data warehouse. REPORT and DATASCIENCE queries read from the larger tpc-h 3t dataset, against which the COPY loads are going in every hour. For the copy we load the tpc-h 100g data files on top of the tpc-h 3t data tables.
+
+Here are the steps to use this benchmark:
+## 1. Cluster Set up: 
+    Create 2 Redshift clusters with different configurations, one as a baseline and one as target
+
+* We have used this to assess the performance between Manual WLM and Auto WLM, here are the WLM configurations we have used:
+
+### Manual Configuration	
+
+Queues/Query Groups	|Memory %	|Max Concurrency	|Concurrency Scaling	|Priority		
+---|----|---|---|---|----|---|---|--
+Dashboard	|24|	5	|Off	|NA	
+Report	|25	|6	|Off	|NA		
+DataScience	|25	|4	|Off	|NA	
+COPY	|25	|3	|Off	|NA	
+Default	|1	|1	|Off	|NA	
+
+### Automatic Configuration
+
+Queues/Query Groups	|Memory %	|Max Concurrency	|Concurrency Scaling	|Priority
+---|----|---|---|---|----|---|---|--
+Dashboard	|Auto	|Auto	|Off	|Normal
+Report	|Auto	|Auto	|Off	|Normal
+DataScience	|Auto	|Auto	|Off	|Normal
+COPY	|Auto	|Auto	|Off	|Normal
+Default	|Auto	|Auto	|Off	|Normal
+
+
+## 2. Metadata Set up: 
+One time Run this set of commands against both clusters. This will set up some users, tables, metadata for the workload run  
+```
+    export PGPASSWORD=<masteruserpassword>
+    psql -h cluster1.ck0e91yaz.us-east-1.redshift.amazonaws.com -p \<portnumber\> -d \<database\> -U \<masteruser\> -f tpc-h-sqls.sql > tpc-h-sqls.sql.manua.log
+    psql -h cluster2.ck0e91yaz.us-east-1.redshift.amazonaws.com -p \<portnumber\> -d \<database\> -U \<masteruser\> -f tpc-h-sqls.sql > tpc-h-sqls.sql-auto.log
+```
+    With sql, you can check if the tables are populated properly. select sql_type  ,  tpch_sql_number from tpc_h_sqls order by 1,2; Should return 28 rows.
+
+
+
+## 3. Initial Data load: 
+One time run this against both clusters. Please note, you have to provide the access key and id for the COPY commands. This steps would take time depending on the instance type. Suggest to run them in back ground.
```

Comment:
Cleanup grammar.  E.g. `Run these once against each cluster... We recommend that you run them in the background.`.

Reply:
fixed.